shims/spark301/src/main/scala/com/nvidia/spark/rapids/shims/spark301/GpuWindowInPandasExec.scala

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
I was a bit surprised to see this class moved and have so much difference from before when it's the 301 shim and not a Databricks thing.  It looks like all Databricks needs to do is override `doColumnar` from the existing base class, but i understand it's not as consistent as what we've done with the other classes where nothing is overridden in the base class by the shim version, as the base class is truly reusable across shim versions.

I could see it either way, but this PR would probably be a lot smaller if we went the "keep 301 where it is" route.  Curious what others think.

Reply:
looking into reverting since it will make review easier and ideally we want to remove 3.0.1 shim anyway.